ui/src/components/option-group/QOptionGroup.json

```
@@ -15,6 +15,30 @@
     "options": {
       "type": "Array",
       "desc": "Array of objects with value and label props. The binary components will be created according to this array; Props from QToggle, QCheckbox or QRadio can also be added as key/value pairs to control the components singularly",
```

```suggestion
      "desc": "Array of objects with value, label, and disable(optional) props. The binary components will be created according to this array; Props from QToggle, QCheckbox or QRadio can also be added as key/value pairs to control the components singularly",
```
Let's clarify the description a bit while we are at it 🙂